packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1923,7 +1922,7 @@ export const createAzureCommunicationCallAdapterFromClient: (callClient: Statefu
 export const createAzureCommunicationCallWithChatAdapter: ({ userId, displayName, credential, endpoint, locator, alternateCallerId, callAdapterOptions }: AzureCommunicationCallWithChatAdapterArgs) => Promise<CallWithChatAdapter>;
 
 // @public
-export const createAzureCommunicationCallWithChatAdapterFromClients: ({ callClient, callAgent, callLocator, chatClient, chatThreadClient }: AzureCommunicationCallWithChatAdapterFromClientArgs) => Promise<CallWithChatAdapter>;
+export function createAzureCommunicationCallWithChatAdapterFromClients(callClient: StatefulCallClient, callAgent: CallAgent, callLocator: CallAdapterLocator | TeamsMeetingLinkLocator, chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient, options?: AzureCommunicationChatAdapterOptions): Promise<CallWithChatAdapter>;
```

Comment:
Wait, I noticed we leaked a breaking change to the beta version...

the parameter should be an object instead of 5 parameters

Reply:
Oh is it suppose to be like this?
```suggestion
export function createAzureCommunicationCallWithChatAdapterFromClients(AzureCommunicationCallWithChatAdapterFromClientArgs, options?: AzureCommunicationChatAdapterOptions): Promise<CallWithChatAdapter>;
```

Reply:
I guess we can still use arrow function cause the param is in an options bag, let's try to make it work, and see if we could either replace our current npm package or push a hot fix